docs/source/cluster_management/clis.rst

```diff
@@ -60,3 +60,7 @@ For example, to run the ``fast-forward`` command with default settings, run ``./
 The ``--offline`` flag will make the CLI ignore the leader, timestamp, and lock configuration blocks, and start an embedded timestamp and lock server.
 Once the CLI has completed, you can resume your AtlasDB clients.
 
+Running Other Commands with an Embedded Server
```

Comment:
Probably should be something like "Running Other Commands without any servers being up". This kinda sounds like you're trying to run commands against an embedded instance.